docs/UsersGuide.asciidoc

```diff
@@ -1021,6 +1021,28 @@ by adding additional query parameters, e.g.:
   `openSUSE Leap 15 ARM`. This applies to parameters for filtering job settings
   as well.
 
+=== Finding tests by quering particular job settings
+
+In case you are looking for a way to get a list of all the tests based on their
+settings, here two ways to achieve it.
+
+[source,sh]
+----
+curl -s "https://openqa.suse.de/api/v1/job_settings/jobs?key=*_TEST_ISSUES&list_value=39911"
+----
+
+or with the power of the `openqa-cli api`
+
+[source,sh]
+----
+openqa-cli api --osd job_settings/jobs key="*_TEST_ISSUES" list_value=39911
+----
+
+It is possible to use a literal string or a glob for C<key> as is shown in the
```

Comment:
This is a bit of duplication to the pod section in the code.
This will improve when we switch to OpenAPI at some point.

Reply:
I didnt change this one. I think this is fine and part of the documentation. or it should be